### PR TITLE
feat: Introduce docs button

### PR DESCRIPTION
**Pull request by faouziH21**
prev:
image
now: 
image

### CODE REVIEW

File: src/components/DocumentationButton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+
```
Comment (faouziH21) [inline src/components/DocumentationButton/index.tsx:1]:
```suggestion

```

File: src/components/DocumentationButton/index.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  className
+}) => {
+
+  const finalClassName = className
+    ? `${defaultClassName} ${className}`
+    : defaultClassName;
+
+  return (
+    <SimpleButton
+      onClick={() => window.open('/gis-docs', '_blank')}
```
Comment (marcjansen) [inline src/components/DocumentationButton/index.tsx:23]: The URL should be configurable, I guess.

File: src/components/Header/index.tsx [hunk 3]
```diff
@@ -97,6 +100,25 @@ export const Header: React.FC<HeaderProps> = ({
...
+        key="documentation-button"
+        aria-label="documentation-button"
+      >
+        <DocumentationButton
+          key="documentation-button"
+          type="link"
+        >
+        </DocumentationButton>
+
+      </div>
```
Comment (marcjansen) [inline src/components/Header/index.tsx:115]: Why do we need a wrapping `<div>`?

File: src/components/Header/index.tsx [hunk 4]
```diff
@@ -97,6 +100,25 @@ export const Header: React.FC<HeaderProps> = ({
...
+    ];
+    insertPlugins('right', items);
+    if (ClientConfiguration.documentationButtonVisible) {
```
Comment (marcjansen) [inline src/components/Header/index.tsx:118]: I'd move the check to the top of the fiunction:

```
if (!ClientConfiguration.documentationButtonVisible) {
  return;
}
```

**Review by marcjansen — COMMENTED** [review]
Just some comments, others should also weigh in

File: src/components/Header/index.tsx (hunk 3 above)
Comment (faouziH21) [reply src/components/Header/index.tsx]: other components used a wrapping `<div>` hence I chose to also wrap it in a div.

File: src/components/Header/index.tsx (hunk 4 above)
Comment (faouziH21) [reply src/components/Header/index.tsx]: check [8861c02](https://github.com/terrestris/shogun-gis-client/pull/1743/commits/8861c02552193e756d49bccfefb9548435aadd86)

**Review by dnlkoch — APPROVED** [review]
Thanks @faouziH21!

**Pull request merged**